plugin/pkg/admission/limitranger/admission.go

```diff
@@ -557,6 +557,7 @@ func PodValidateLimitFunc(limitRange *corev1.LimitRange, pod *api.Pod) error {
 					}
 				}
 			}
+			continue
```

Comment:
limitType is either LimitTypeContainer or corev1.LimitTypePod, why is this 'continue' necessary?

Reply:
@yliaog  thanks for your comments.

Not sure which is better.  What I want to do is like below. 

```
	for i := range limitRange.Spec.Limits {
		limit := limitRange.Spec.Limits[i]	
		limitType := limit.Type	
		// enforce container limits	
		if limitType == corev1.LimitTypeContainer {
			...
+			continue //directly to next loop;skip the next if
		} 
		if limitType == corev1.LimitTypePod {
			
		}
	}
```

or

```
	for i := range limitRange.Spec.Limits {
		limit := limitRange.Spec.Limits[i]	
		limitType := limit.Type	
		// enforce container limits	
		if limitType == corev1.LimitTypeContainer {
			...
			continue
		} 
+		else if limitType == corev1.LimitTypePod {
-		if limitType == corev1.LimitTypePod {
			
		}
	}
```

Not sure if this is needed or makes sense.

However, this is not the key part of this PR.

Reply:
I think `else if` is better in this case. So I changed it.

Reply:
@yliaog  As I have another bugfix pr #98295 for limit range, I move the cleanup code to that PR.
So the code owner could focus on the single package.

Thanks for your comments anyway.